integrated_channels/integrated_channel/transmitters/learner_data.py

```diff
@@ -134,6 +134,7 @@ def single_learner_assessment_grade_transmit(self, exporter, **kwargs):
 
             learner_data.status = str(code)
             learner_data.error_message = body if code >= 400 else ''
+            learner_data.is_transmitted = code < 400
```

Comment:
[Question] Have we checked how will this statement behave in case code is None or empty string?

Reply:
It's taken from the line above it.... which works fine for us 🤷